_po4a-tools/po4a-update-templates.sh

```diff
@@ -46,15 +51,15 @@ fi
 
 while IFS= read -r -d '' file ; do
     # Determine target file/folder names
-    basename=$(basename -s .md "$file")
+    basename="$(basename -s .md "$file")"
```

Comment:
Not sure about the quoting here.

Reply:
I'm pretty sure I was asked to add them to the same expression in create-all-targets, (by @pljones perhaps? not sure), so added them here too. But I could also be mistaken. The scripts work though.

Reply:
They do probably work. Syntactically I‘m not sure if they’re correct.

Reply:
So I went back to see if I'd been told to add those quotes, and it doesn't seem like it... so I removed them from both scripts.